rosidl_typesupport_connext_cpp/cmake/rosidl_typesupport_connext_cpp_generate_interfaces.cmake

```diff
@@ -61,11 +61,21 @@ endforeach()
 
 # If not on Windows, disable some warnings with Connext's generated code
 if(NOT WIN32)
-  set(_connext_compile_flags
-    "-Wno-tautological-compare"
-    "-Wno-return-type-c-linkage"
-    "-Wno-deprecated-register"
-  )
+  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    set(_connext_compile_flags
+      "-Wno-deprecated-register "
+      "-Wno-mismatched-tags "
+      "-Wno-return-type-c-linkage "
+      "-Wno-sometimes-uninitialized "
+      "-Wno-tautological-compare "
+      "-Wno-unused-variable "
```

Comment:
The string values should not have a trailing space inside. Same below for the GNU flags.